helm/sealed-secrets/crds/sealedsecret-crd.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: sealedsecrets.bitnami.com
+spec:
+  group: bitnami.com
+  names:
+    kind: SealedSecret
+    listKind: SealedSecretList
+    plural: sealedsecrets
+    singular: sealedsecret
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    served: true
+    storage: true
+    subresources:
+      status: {}
+    schema:
+      openAPIV3Schema:
+        type: object
+        properties:
+          spec:
+            type: object
+            x-kubernetes-preserve-unknown-fields: true
+            properties: {}
```

Comment:
This line is causing pain for argocd because empty things are odd...

![image](https://user-images.githubusercontent.com/2119212/107210763-95128680-69d2-11eb-84df-f730b919b33d.png)

Reply:
What do you suggest?

Reply:
First, please note that I'm by no means an expert (I'm unfortunately bumping into all of these things more or less at once).

Can you explain to me what you're trying to say by having this field?

Are you saying that properties is mandatory but can have no items?

For your field, would this be valid:
```yaml
properties: null
```

I'm assuming you're saying that's invalid.

And that you only want:

```yaml
properties:
  ...
```

---
It's possible that something like:

```yaml
properties:
  spec:
    properties:
      type: object
```
might do what you want. It's also possible I'm totally wrong :-(
I'm picking that syntax from https://github.com/kubernetes/apiextensions-apiserver/issues/25#issuecomment-755099112

It's possible that you want:
```yaml
properties:
  spec:
    properties:
      type: object
      nullable: true
```